scality_manila_utils/utils.py

```diff
@@ -119,6 +119,22 @@ def process_check(process):
                                    "started".format(process))
 
 
+def fsync_path(path):
+    """
+    Fsync a file or directory.
+
+    :param path: path to file or directory to fsync
+    :type path: string (unicode)
+    """
+    fd = None
+    try:
+        fd = os.open(path, os.O_RDONLY)
```

Comment:
Good point, I agree that the writer of a file should ensure any fsyncing. I will change it accordingly.